rpc/block_svc_routines.c

```diff
@@ -1450,6 +1611,17 @@ glusterBlockBuildMinCaps(void *data, operations opt)
       minCaps[GB_JSON_CAP] = true;
     }
     break;
+  case MODIFY_SIZE_SRV:
+    msblk = (blockModifySizeCli *)data;
+
+    minCaps[GB_MODIFY_CAP] = true;
```

Comment:
We can remove the line above, considering there is no relation between these two capabilities now.